index/mapping.py

```diff
@@ -2,16 +2,19 @@
 import numpy as np
 
 from index.embedding import EmbeddingModel
-from index.db.model import Terminology, Mapping, Concept, Variable
```

Comment:
Here might be an issue - the model file was here moved to the db package. Weird that this does not throw an error in the parser test or python package though, I will have to take a look at that.

Could you adjust this?